README.md

```diff
@@ -10,9 +10,9 @@ calls. With GPTScript you can do just about anything, like [plan a vacation](./e
 [edit a file](./examples/add-go-mod-dep.gpt), [run some SQL](./examples/sqlite-download.gpt), or [build a mongodb/flask app](./examples/hacker-news-headlines.gpt).
 
 | :memo: | We are currently exploring options for interacting with local models using GPTScript. |
-|-|:-|
+| ------ | :------------------------------------------------------------------------------------ |
 
-```yaml
+````yaml
```

Comment:
This line has an extra backtick, and a couple others do too

Reply:
Fun fact, this was Zed's auto-formatting 🙃

Reply:
Oh lol